app/components/tab_bar/tab_bar.css

```diff
@@ -0,0 +1,36 @@
+.TabBar {
+  list-style: none;
+}
+
+.TabBar-item {
+  display: inline-block;
+  padding: var(--spacing-unit-xs) 0;
+  border-bottom: 2px solid transparent;
+  color: var(--color-text-light);
+  font-weight: 500;
+  transition: 0.1s all ease-out;
+}
+
+.TabBar-item + .TabBar-item {
+  margin-left: var(--spacing-unit-s);
+}
+
+.TabBar-item:not(.TabBar-item--active):hover {
+  border-bottom-color: var(--color-gray);
+  color: var(--color-text);
+}
+
+.TabBar-item--active {
+  border-bottom-color: var(--color-blue);
+  color: var(--color-text);
+}
+
+.TabBar-count {
+  display: inline-block;
+  margin-left: 0.25em;
+  padding: 0 0.75em;
+  background-color: var(--color-gray);
+  color: #fff;
+  font-size: var(--font-size-xs);
+  border-radius: 9999px;
```

Comment:
You could fix that after https://github.com/tactilenews/100eyes/pull/1033

Reply:
Yes, will do that!